File: CS/IntAsEnumExample.Module.Web/Editors/MyEnumIntPropertyEditor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp.Editors;
using DevExpress.ExpressApp.Web.Editors.ASPx;
using IntAsEnumExample.Module.BusinessObjects;
using DevExpress.ExpressApp.Model;

namespace IntAsEnumExample.Module.Web.Editors {
    [PropertyEditor(typeof(int), false)]
    public class ASPxMyEnumIntPropertyEditor : ASPxEnumIntPropertyEditor<SampleEnum> {
        public ASPxMyEnumIntPropertyEditor(Type objectType, IModelMemberViewItem model)
            : base(objectType, model) {
        }
    }
}

File: VB/IntAsEnumExample.Module.Web/Editors/MyEnumIntPropertyEditor.vb
Imports System
Imports DevExpress.ExpressApp.Editors
Imports DevExpress.ExpressApp.Web.Editors.ASPx
Imports IntAsEnumExample.Module.BusinessObjects
Imports DevExpress.ExpressApp.Model

Namespace IntAsEnumExample.Module.Web.Editors

    <PropertyEditor(GetType(Integer), False)>
    Public Class ASPxMyEnumIntPropertyEditor
        Inherits ASPxEnumIntPropertyEditor(Of SampleEnum)

        Public Sub New(ByVal objectType As Type, ByVal model As IModelMemberViewItem)
            MyBase.New(objectType, model)
        End Sub
    End Class
End Namespace
